catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.config.xml

```diff
@@ -201,6 +201,13 @@
             default=""
             required="true"/>
 
+        <AD id="maximumUploadSize"
+            name="Maximum Endpoint Upload Size"
+            description="The maximum size (in bytes) to allow per client when receiving a POST/PATCH/PUT. Note: This does not affect product upload size, just the maximum size allowed for calls from Intrigue"
+            type="Integer"
```

Comment:
This is not a product upload, just the endpoint request size limit (usually sending a json blob, etc). I will rename this to `maxRequestSize` to make it a little clearer that its not a product upload.